package.json

```diff
@@ -14,10 +14,11 @@
   "license": "MIT",
   "homepage": "https://github.com/Synthetixio/synthetix-v3#readme",
   "scripts": {
+    "build": "npm run --workspaces --if-present build",
     "test": "npm run --workspaces --if-present test",
     "coverage": "npm run --workspaces --if-present coverage",
-    "lint:js": "prettier --check '**/*.js' && eslint '**/*.js'",
-    "lint:js:fix": "prettier --write '**/*.js' && eslint --fix '**/*.js'",
+    "lint:js": "eslint '**/*.ts'",
+    "lint:js:fix": "eslint --fix '**/*.ts'",
```

Comment:
Why are we removing prettier here?

Reply:
it was causing errors for "warnings" in typescript code which can't be handled right now and, if i'm not mistaken, eslint covers prettier problems already so its redundant.